panel/scripts.py

```diff
@@ -35,3 +40,36 @@ def apply_dec(vac_edit_obj, decision):
         else:
             vac_edit_obj.delete()
             return 'deleted'
+
+
+def count_vac_days(start_date, end_date):
+    holidays = []
+    start_date_obj = time.strptime(start_date, '%Y-%m-%d')
```

Comment:
Yep. You're right. Not caring about date format is the main point i think. Function will be much more reusable